### ISSUE TITLE
Latest version has weird string

### ISSUE THREAD

**Issue by marcusboon** [vasqued2/ha-teamtracker-card#58]
Updated to 0.8.0 and noticed that for a custom integration (rugby league in Australia) instead of the regular “Kickoff in X timeframe” it’s now `{rugby-league.startTerm}` in X timeframe. 

![image](https://github.com/vasqued2/ha-teamtracker-card/assets/3231180/d5d1751f-0be9-4f4f-99ac-544169d66278)

The integration looks something like this:
```
- platform: teamtracker
  league_id: XXX
  team_id: “*”
  sport_path: rugby-league
  league_path: 3
  name: NRL
```

**Comment by vasqued2** [comment]
Thanks.  Should be able to get a fix out this weekend.

**Comment by vasqued2** [comment]
Should be fixed in v0.8.1

**Comment by marcusboon** [comment]
Hey @vasqued2 looking forward to it! Noticed some other string translation issues

![image](https://github.com/vasqued2/ha-teamtracker-card/assets/3231180/aa7e6831-51da-45fa-b0a9-22b71204459b)

Would those be fixed in 0.8.1 as well?

**Comment by marcusboon** [comment]
Never mind, just updated and can see that it is indeed fixed. Thanks once again!

![image](https://github.com/vasqued2/ha-teamtracker-card/assets/3231180/2ccb8cf8-7459-4b4e-a200-9c9f2d936d4c)
